dev-client/src/components/form/FormFieldWrapper.tsx

```diff
@@ -18,27 +18,33 @@
 import {memo} from 'react';
 import {FormField} from 'terraso-mobile-client/components/form/FormField';
 import {FormLabel} from 'terraso-mobile-client/components/form/FormLabel';
-import {ErrorMessage as FormikErrorMessage, useFormikContext} from 'formik';
+import {ErrorMessage as FormikErrorMessage} from 'formik';
 import {FormControl} from 'native-base';
 import {useFieldContext} from 'terraso-mobile-client/components/form/hooks/useFieldContext';
 
 const FormHelperText = memo(
-  (props: React.ComponentProps<typeof FormControl.HelperText>) =>
-    useFieldContext().name! in useFormikContext().errors ? undefined : (
+  (props: React.ComponentProps<typeof FormControl.HelperText>) => {
+    return useFieldContext().error ? undefined : (
```

Comment:
question: Does this fix it so that the HelperText is displayed when there is no error? I was running into issues with this in something I was working on too and ended up adding it separately. If this fixes it I should change it in the PR.

Reply:
i actually should take these changes out of the PR, they were for a part of the screen i ended up not needing. it was for a situation where i wanted to use our form components without a formik context but i think that's broken with these changes and these abstractions probably need some love generally